nixos/lib/default.nix

```diff
@@ -1,6 +1,8 @@
 let
-  # The warning is in a top-level let binding so it is only printed once.
+  # The warnings are in a top-level let binding so it is only printed once.
   minimalModulesWarning = warn "lib.nixos.evalModules is experimental and subject to change. See nixos/lib/default.nix" null;
+  standardModulesWarning = warn "lib.nixos.evalSystem is experimental and subject to change. See nixos/lib/default.nix" null;
```

Comment:
Unlike minimal, this one should be possible to stabilize soon-ish.

Reply:
let's make it public & stable (as pristine as it is) straight away.